src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,169 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * A class that handles all the user interactions of the program
+ */
+public class Ui {
+    private String lnspace = "____________________________________________________________";
```

Comment:
You could also try making this final and changing the variable name to LNSPACE. But this works too!